client/app/_components/ui/dialog/InvitedHouseAlertDialog.tsx

```diff
@@ -0,0 +1,36 @@
+'use client'
+
+import { useState } from 'react'
+import { FiAlertTriangle } from 'react-icons/fi'
+import { Dialog, DialogContent, DialogTitle } from '@/_components/ui/Dialog'
+import { useCheckInvitedHouse } from '@/_hooks'
+
+function InvitedHouseAlertDialog() {
+  const [isOpen, setIsOpen] = useState(false)
+
+  const onClose = () => setIsOpen(false)
+  const onOpen = () => setIsOpen(true)
+
+  useCheckInvitedHouse(onOpen)
+
+  return (
+    <Dialog open={isOpen} onOpenChange={onClose}>
+      <DialogContent className=''>
```

Comment:
You can delete this empty className!

Reply:
Thanks! I deleted it https://github.com/HAK2024/fairy-share/pull/76/commits/05b6b392d06941a54e91aa5e1033c01d389d418c